lm_eval/api/utils.py

```diff
@@ -18,17 +18,17 @@ class ExitCodeError(Exception):
 # Reproducibility utils
 
 
-def get_seed() -> int:
-    """Returns a hard-coded global seed for reproducibility."""
+def get_default_seed() -> int:
+    """Returns a hard-coded global default seed for reproducibility."""
     return 1234
 
 
-def set_seed() -> None:
-    transformers_set_seed(get_seed())
+def set_seed(seed: Optional[int] = get_default_seed()):
+    transformers_set_seed(seed)
 
 
-def get_rng() -> numpy.random.Generator:
-    return numpy.random.default_rng(get_seed())
+def get_rng(seed: int) -> numpy.random.Generator:
```

Comment:
Since `get_rng` seems to be only used in one place, do we need this extra layer of abstraction?

Reply:
Good point. Thanks!